src/Kunstmaan/UtilitiesBundle/Helper/Shell/Shell.php

```diff
@@ -10,19 +12,21 @@ class Shell implements ShellInterface
 
     /**
      * @param string $command  The command
-     * @param int    $priority The priority
+     * @param int|null    $priority The priority
      *
      * @return string The process id
      */
     public function runInBackground($command, $priority = 0)
     {
-        if ($priority) {
-            $pid = shell_exec("nohup nice -n $priority $command > /dev/null & echo $!");
-        } else {
-            $pid = shell_exec("nohup $command > /dev/null & echo $!");
-        }
+        $command = null !== $priority ?
```

Comment:
As said in my other comment we should be able to drop the nohup from here especially if we are using Process->start() then it is already running in background. 
Also we can also disable output using the Process object, that way we don't have to write to /dev/null ourselves either.